### PR TITLE
added text fixing suggestions

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: smt_fv.tex [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+We translate Solidity contracts and their functions into SMT formulas using a
+combination of different quantifier-free theories.
+%
+We shall name the translated formulas the \emph{SMT encoding} of the Solidity
+program.
+%
+The goal of the translation from Solidity to SMT formulas is to verify safety
+properties from the Solidity program by performing queries to the SMT solver.
+
 \todo{the transition from the previous section to
```
[the remark this answers is not in the document]
Comment (chriseth) [reply smt_fv.tex]: great!

File: smt_fv.tex [hunk 2]
```diff
@@ -69,8 +84,9 @@ \section{SMT Encoding}
 Since the variables \code{uint256 a} and \code{uint256 b} are function parameters, they
 are initialized with the valid range of values for their type (\code{uint256}).
 %
-The requirement condition about \code{b} is only true if the first condition
-about \code{a} is true \todo{C: I do not understand this}.
+The \code{require} condition about \code{b} should only be taken into account
+if \code{a = 0}, which is expressed in the logical implication. \todo{C: I do
+not understand this; L: maybe better now?}.
```
[the remark this answers is not in the document]
Comment (chriseth) [reply smt_fv.tex]: yep!

File: smt_fv.tex [hunk 3]
```diff
@@ -91,7 +107,8 @@ \section{SMT Encoding}
 \begin{verbatim}
 contract C
 {
-  function f(uint256 a, uint256 b) {
```
Comment (chriseth) [inline smt_fv.tex]: I think we can use the more compact form, but I'm also not sure.

File: smt_fv.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (chriseth) [reply smt_fv.tex]: But actually still - what exactly is encoded here? We should explain why the require matters when we actually care about the assert in the end.

File: smt_fv.tex (hunk 3 above)
Comment (leonardoalt) [reply smt_fv.tex]: The `{` was kinda close to the formula, that's why I sent it down

File: smt_fv.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (leonardoalt) [reply smt_fv.tex]: Changed the text

**Pull request merged**